src/workspaces/features/RunPipelineDialog/ParameterField.tsx

```diff
@@ -39,6 +42,18 @@ const ParameterField = (props: ParameterFieldProps) => {
       />
     );
   }
+
+  if (isConnectionParameter(parameter.type)) {
+    return (
+      <WorkspaceConnectionPicker
+        workspaceSlug={workspaceSlug || ""}
```

Comment:
Can `workspaceSlug` be null or undefined?

Reply:
On the model yes.